includes/tree.html

```diff
@@ -1,14 +1,14 @@
 <div ui-tree data-nodrop-enabled="true" data-drag-enabled="false" id="context-tree">
     <ul ui-tree-nodes ng-model="contexts" style="margin-bottom: 8px;">
-        <li class="clickable" ng-click="createNewContext()">
-            <i class="material-icons placeholder"></i><i class="material-icons md-18">add</i>
+        <li class="btn btn-primary clickable" ng-click="createNewContext()">
```

Comment:
as discussed, this element triggers a data changing action and should therefore be button styled